devops/windows/base-docker-wsl/helper/log/logs-lambda.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -e
+set -u
+
+source 0-conf.env
+docker service logs --raw ${STACK_NAME}_lambda
```

Comment:
What is the lambda service?

Reply:
lambda is a dgraph extension --- can be deleted until we really this